### PR TITLE
Updated the affiliations of Marc Toussaint and Kostas Alexis.

### PR DESCRIPTION
**Pull request by adityamwagh**
Marc Toussaint has moved to TU Berlin from University of Stuttgart.

### CODE REVIEW

**Comment by adityamwagh** [comment]
Kostas Alexis has moved to NTNU from University of Nevada, Reno.  The commit  `e0588d2` represents that change.

File: csrankings-4.csv [hunk 1]
```diff
@@ -2063,7 +2063,7 @@ Korin Richmond,University of Edinburgh,http://homepages.inf.ed.ac.uk/korin,C_VIk
 Korlakai Vinayak Rashmi,Carnegie Mellon University,http://www.cs.cmu.edu/~rvinayak,gUO59T8AAAAJ
 Kostadin Damevski,Virginia Commonwealth University,http://damevski.github.io,3f847ugAAAAJ
 Kostantinos Kalpakis,Univ. of Maryland - Baltimore County,http://www.csee.umbc.edu/people/faculty/konstantinos-kalpakis,NOSCHOLARPAGE
-Kostas Alexis,University of Nevada,https://www.unr.edu/cse/people/faculty/alexis,1rPSVNAAAAAJ
+Kostas Alexis,NTNU,http://www.kostasalexis.com/
```
Comment (emeryberger) [inline csrankings-4.csv]: Missing Google Scholar link.

File: csrankings-5.csv [hunk 2]
```diff
@@ -1039,7 +1039,7 @@ Marc Stamminger,University of Erlangen–Nuremberg,https://www.lgdv.tf.fau.de/pe
 Marc Stevens,CWI,https://marc-stevens.nl,Zhnqx9IAAAAJ
 Marc Streit,JKU Linz,https://www.jku.at/en/institute-of-computer-graphics/about-us/team/marc-streit,l7TTSP0AAAAJ
 Marc Tommasi,CRIStAL,http://researchers.lille.inria.fr/tommasi,IRyM3b8AAAAJ
-Marc Toussaint,University of Stuttgart,http://ipvs.informatik.uni-stuttgart.de/mlr/marc,t2X4Mg8AAAAJ
+Marc Toussaint,TU Berlin,https://www.user.tu-berlin.de/mtoussai/index.html
```
Comment (emeryberger) [inline csrankings-5.csv]: Missing Google Scholar link.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-5.csv (hunk 2 above)
Comment (adityamwagh) [reply csrankings-5.csv]: Missed it! Updating ASAP.

File: csrankings-4.csv (hunk 1 above)
Comment (adityamwagh) [reply csrankings-4.csv]: Missed it! Updating ASAP.

**Pull request merged**